src/UnderlineNav2/styles.ts

```diff
@@ -142,3 +145,17 @@ export const menuItemStyles = {
   // To reset the style when the menu items are rendered as react router links
   textDecoration: 'none'
 }
+
+export const getMenuStyles = (theme?: Theme, isWidgetOpen?: boolean) => ({
+  position: 'absolute',
+  top: '90%',
+  right: '0',
+  boxShadow: '0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24)',
+  borderRadius: '12px',
+  backgroundColor: `${theme?.colors.canvas.overlay}`,
```

Comment:
We probably don't need to access this from theme and can replace with the token name

```suggestion
backgroundColor: 'canvas.overlay'
```

Reply:
Oh I didn't know I can just pass the token name! Thank you 🙌🏼  Is it only for colours as we import only colour primitives currently maybe? I see mix practises around, where can I learn more about this? 👀 or should I just dig in? 😅

Reply:
I'll do a clean up to remove the other unnecessary `theme` args in the next PR!